var/spack/repos/builtin/packages/qthreads/package.py

```diff
@@ -85,3 +85,58 @@ def configure_args(self):
                     % self.spec.variants['scheduler'].value)
 
         return args
+
+
+
```

Comment:
```suggestion
    test_requires_compiler = True
```

Reply:
You'll need this property set to `True` to get the right compiler for the build.